sound/soc/intel/boards/sdw_rt711_rt1308_rt715.c

```diff
@@ -579,36 +582,62 @@ static inline int find_codec_info_acpi(const u8 *acpi_id)
 	return i;
 }
 
-static int get_sdw_dai_link_num(struct snd_soc_acpi_mach_params *mach_params)
+/*
+ * get BE dailink number and CPU DAI number based on sdw link adr.
+ * Since some sdw links may be aggregated, the CPU DAI number may be
```

Comment:
links are aggregated? Or slaves are aggregated?

Reply:
links are also aggregated. one PCM data will be sent to both links to support this PCM.